drivers/timer/hpet.c

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
change you commit title to "drivers: timer: Enable 64 bit mode in HPET for better usage" or alike

Reply:
done.